changelog.d/pa-3055.fixed

```diff
@@ -0,0 +1 @@
+The CFG now supports case statements in Ruby, which does not fall through.
```

Comment:
This entry is not very good TBH, I do not think any user will understand this. For users it's confusing already because you talk about CFGs. Why not something like "Semgrep now handles Ruby case statements correctly and does not allow control to fall-through", and then you put a code snippet e.g. with `source` and `sink` that shows what now works correctly.

Reply:
Sounds good. Can I still do this after it's merged, but before the release?

Reply:
For sure, the actual changelog is only generated at release time. If you want to fix this I really appreciate it, but my intention was that you just considered this for the next time. 🙏